.github/labeler.yml

```diff
@@ -21,6 +21,14 @@
         - pkgs/by-name/ne/nemo/**/*
         - pkgs/by-name/ne/nemo-*/**/*
 
+"6.topic: darwin":
```

Comment:
I'd suggest removing the darwin label from ofborg if it is going to be added here, ofborg adds it based on the PR title but the action will remove labels that don't have a path match.

Reply:
Oh, that’s a shame. Do you know if there’s any way to get both of them to cooperate? Maybe we should just drop this commit if not, although I think ofborg’s heuristic isn’t particularly great.

Reply:
Thanks. I’ve dropped the labeller commit for now.